File: Onboarding/two-sum.js
/**
 * @param {number[]} number
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(number, target) {
    for (var i = 0; i < number.length; i++){
        for (var j = 0; j < number.length; j++){
            if ( number[i] + number[j] == target){
                if (i != j ){
                    return [i , j ];
                }  

            }

        }
    } return [-1]
    
};

File: Onboarding/two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        l, r = 0 , len(nums) - 1
        store = {}
        for ind, val in enumerate(nums):
            if target - val in store:
                return [store[target-val],ind]
            store[val] = ind
